.github/workflows/aws.yml

```diff
@@ -0,0 +1,128 @@
+# Basic workflow with Actions
+
+name: AWS Cloud Database Test Execution
```

Comment:
@jnewton03 -- are we tied to only using this for AWS? Can we use the same yaml for other cloud platforms like snowflake? Or would you then want to put that in a separate `snowflake.yml` file?

Reply:
@kristyldatical I wanted to be able to test without doing the whole docker workflow, so I created a different action.  I think we could use the same workflow for other clouds.  My thought was to use a single job that runs Terraform to provision the infrastructure and then the test job would run the matrix to run the tests in parallel.  Would love to get @mcred 's thoughts on that.